pkg/kubectl/cmd/testing/fake.go

```diff
@@ -701,5 +701,21 @@ func testDynamicResources() []*discovery.APIGroupResources {
 				},
 			},
 		},
+		{
+			Group: metav1.APIGroup{
+				Name: "unit-test.test.com",
+				Versions: []metav1.GroupVersionForDiscovery{
+					{GroupVersion: "unit-test.test.com/v1", Version: "v1"},
+				},
+				PreferredVersion: metav1.GroupVersionForDiscovery{
+					GroupVersion: "unit-test.test.com/v1",
+					Version:      "v1beta1"},
+			},
+			VersionedResources: map[string][]metav1.APIResource{
+				"v1": {
+					{Name: "testtprs", Namespaced: true, Kind: "TestTpr"},
```

Comment:
this is not TPR-specific, prefer `{Name: "widgets", Namespaced: true, Kind: "Widget"}` or something